kotishell/src/tokenize/tokenize_c.c

```diff
@@ -25,3 +25,38 @@ t_tk_state	tk_next_state(t_tk_state state, char *s)
 		state = TK_ST_WORD;
 	return (state);
 }
+
+void	s_tk_print_list(t_li_node *list)
```

Comment:
Could make sense to move to the test directory

Reply:
done